examples/input/multiple_examples_per_slot.tsv

```diff
@@ -0,0 +1,6 @@
+class	slot	description	examples
+> class	slot	description	examples
+>			"internal_separator: ""|"""
```

Comment:
is the additional quoting intentional? I forget how csvreader handles this

Reply:
not intentional. removed in most recent commit. still works.